self-tests.yml

```diff
@@ -421,6 +421,13 @@ functions:
           set -o xtrace
           rm evergreen/bin/output.*
           rm evergreen/bin/jstests/*.xml
+  
+  write-repo-name-to-downstream_expansions.yaml-for-pine:
+    - command: shell.exec
+      params: 
+        script: |
+          touch downstream_expansions.yaml
+          echo "pine_repo_name: evergreen" | tee downstream_expansions.yaml
```

Comment:
same comment as below, let's mention that its for pine in the yaml name

Reply:
I'm not sure if changing the yaml name will still make it work (was following Amado's instructions here:  https://docs.devprod.prod.corp.mongodb.com/pine/onboarding/#incorporating-pines-evergreen-task).

Reply:
the name of the file won't affect the command as long as you use the same name to call the `downstream_expansion.set` command below